src/pages/new/volunteer/index.js

```diff
@@ -92,6 +92,7 @@ export default () => (
         },
       ]}
     />
+    <Spacer heightDesktop="120px" heightMobile="80px" />
```

Comment:
I think this might need to be 190px in desktop? Since the image that hangs off the bottom of the component above doesn't seem to accounted for as part of the spacing.

Reply:
ok